src/commands/dynamic_voice.ts

```diff
@@ -131,13 +131,14 @@ export const useDynamicVoice = (): Command => {
           new ButtonBuilder().setCustomId('c-dynamic-voice-increment').setLabel('+').setStyle(ButtonStyle.Success)
         )
 
-      await voice
-        .send({
-          content: `<@${member.id}> aqui está o seu novo canal de voz!`,
-          embeds: [embed],
-          components: [component],
-        })
-        .catch(() => { })
+      const message = {
+        content: `<@${member.id}> aqui está o seu novo canal de voz!`,
+        embeds: [embed],
+        components: [component],
+      }
+
+      await member.send(message).catch(() => {})

```

Comment:
I'll try later, but I think this would be the correct way:

```suggestion
await openAndSendMessageInDm(client, member, message)
```

https://github.com/he4rt/he4rt-bot-next/blob/main/src/utils.ts#L237

> As @anibalsolon said, the DM will have to be destroyed. By default, discord destroys after a certain amount of idle time.

Reply:
> For note: at this time, discord does not allow pinning messages to voice channels with a built-in message channel.